Ddaangn/src/main/java/com/dev/ddaangn/image/service/ImageService.java

```diff
@@ -59,11 +59,9 @@ public ImageDto findOneById(Long id) throws NotFoundException {
 
     @Transactional
     public void deleteOneById(Long id) throws NotFoundException {
-        if(imageRepository.findAll().isEmpty()){
-            throw new NotFoundException(ErrorMessage.NOT_EXIST_IMAGE.message());
+        if (imageRepository.findAll().isEmpty()) {
```

Comment:
```suggestion
        if (imageRepository.existsById(id)) { 
```
를 사용하면 코드 가독성과 의도에 맞을  것 같습니다.